src/fping.c

```diff
@@ -248,7 +248,7 @@ unsigned int interval = DEFAULT_INTERVAL * 100;
 unsigned int perhost_interval = DEFAULT_PERHOST_INTERVAL * 100;
 float backoff = DEFAULT_BACKOFF_FACTOR;
 unsigned int ping_data_size = DEFAULT_PING_DATA_SIZE;
-unsigned int count = 1;
+unsigned int count = 1, reachable = 1;
```

Comment:
I suggest to rename "reachable" to something more descriptive like "min_reachable". Also, the default should be 0?

```suggestion
unsigned int count = 1, min_reachable = 0;
```

Reply:
Changed the initialisation.